pages/common/shiori.md

```diff
@@ -0,0 +1,24 @@
+# shiori
+
+> Simple bookmark manager built with Go.
+> More information: <https://github.com/go-shiori/shiori>.
+
+- Import bookmarks from HTML Netscape bookmark format file:
+
+`shiori import {{path/to/bookmarks.html}}`
+
+- Save the specified URL as bookmark:
+
+`shiori add {{url}}`
+
+- List the saved bookmarks:
+
+`shiori print`
+
+- Open the saved bookmark in a browser:
+
+`shiori open {{bookmark_id}}`
+
+- Start the web interface for managing bookmarks at port 8181:
+
+`shiori serve -p {{8181}}`
```

Comment:
Is there a long for of this `-p` option? In tldr-pages we prefer long options over short ones for clarity.

Reply:
sure, I'll fix it in a couple of minutes.